build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-	id("uk.gov.justice.hmpps.gradle-spring-boot") version "1.0.2"
+	id("uk.gov.justice.hmpps.gradle-spring-boot") version "1.0.4"
 	kotlin("plugin.spring") version "1.4.0"
 	kotlin("plugin.jpa") version "1.4.0"
 	kotlin("plugin.allopen") version "1.4.0"
```

Comment:
I don't think you need this if you have the spring plugin - it opens up all the relevant spring classes

Reply:
Ah sorry ignore me,  just seen the javax.persistence stuff :)